js/dominion.js

```diff
@@ -833,7 +833,8 @@
 
   $(function ()
     {
-      var $advancedSearch = $('body.informationobject.browse');
+      // Find search for if on an appropriate page
+      var $advancedSearch = $('body.informationobject.browse').add('body.search.descriptionUpdates');
```

Comment:
I think that a comma-separated list of selectors will be faster because only one jQuery object is built. The docs of add() give more details.